splunk_add_on_ucc_framework/ucc_ui_lib/src/main/webapp/util/script.js

```diff
@@ -12,18 +14,10 @@ export function getBuildDirPath() {
     return '';
 }
 
-// NOTE: The resolve will only be executed if the globalConfig exist
 export function loadGlobalConfig() {
     // Get the configuraiton json file in sync mode
-    return new Promise((resolve, reject) => {
-        fetch(`${getBuildDirPath()}/globalConfig.json`).then((res) => {
-            return res.json();     
-        }).then((json) => {
-            // window.__globalConfig = json;
-            resolve(json);
-        }).catch((err) => {
-            reject(err);
-        });
+    return axios.get(`${getBuildDirPath()}/globalConfig.json`).then((res) => {
+        return typeof res.data === 'object' ? res.data : JSON.parse(res.data);
```

Comment:
I think `axios` already performs JSON parsing based on response `Content-Type` header. Please verify and remove it if not required.

Reply:
I verified that, 
but when if there is any syntax error in the JSON response, it returns the response in the string which won't return any error until we validate it.